.github/workflows/continuous-integration.yml

```diff
@@ -73,8 +73,10 @@ jobs:
             exit 0
           fi
 
+          pip install osu-wiki-tools
+
           ARTICLES=$( git diff --diff-filter=d --name-only ${{ github.sha }}^ ${{ github.sha }} "wiki/**/*.md" "news/*.md" )
-          python scripts/ci/find_broken_wikilinks.py --target $ARTICLES
+          find-broken-wikilinks --target $ARTICLES
```

Comment:
it looks like all the scripts from the package are added to `PATH`. I feel like making a small CLI module that would proxy the arguments and run necessary scripts is better for clarity and usability. something like that (in addition to the way the scripts are invoked right now):

```python
wikitools-cli check-links --target ... --some-other-flag=...
```

please keep in mind that it's not a requirement for the PR to be merged, but something to do to the tools collection anyway, separately